config/application.rb

```diff
@@ -8,6 +8,11 @@
 
 module Autolab3
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    # config.load_defaults 5.0
```

Comment:
Technically, we should strive to flip over to the new Rails defaults eventually by setting `config.load_defaults 5.0` or even `6.0`. However, some existing code breaks (e.g. the new default `active_record.belongs_to_required_by_default = true` causes issues with current code that does not necessarily initialize foreign keys). This isn't necessary per se so it is fine to continue using old defaults for now.